docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3202,23 +3202,57 @@ are explained in more depth in the table below.
 
 **OCR**
 
-Optical character recognition is when a character is misread for another character that 
-looks similar. A common examples is 'S' and '5'. In order to emulate 
-that, there is a `GeCo like corrupter and related list of possible changes in the ocr_variations_upper_lower csv found here <https://github.com/ihmeuw/vivarium_research_prl/tree/main/src/vivarium_research_prl/noise>`_. 
+Optical character recognition is when a string is misread for another string that 
+is visually similar. Some common examples are 'S' instead of '5' and 'm' instead of 'iii'.
 
-To implement this, select the strings eligible for noise and apply 
-the OCR noise function to all strings with the user defined token 
-error rate. 
+.. todo::
+  Both of these examples seem like they might be backward, though they come from the CSV below.
+  Are we interpreting the CSV backward? Should it be commutative?
+
+We have compiled `a list of possible OCR substitutions in a CSV file <https://github.com/ihmeuw/vivarium_research_prl/blob/main/src/vivarium_research_prl/noise/ocr-variations-upper-lower.csv>`,
```

Comment:
this comes from GeCO, right?

Reply:
@aflaxman I got these .csv files from you. Where did you get them? Do you have a link to the source of the data?